geoutils/geoviewer.py

```diff
@@ -140,7 +140,8 @@ def main() -> None:
         new_shape = None
 
     # Read image #
-    img.load(out_shape=new_shape)
+    img.load()
+    img = img.reproject(dst_size=new_shape)  # type: ignore
```

Comment:
Just a note here. This sounds like one needs to load the whole array before being able to reproject, particuarly downsample it?
If so, this should be improved at some point.

Reply:
This is the case in the current main as well. I agree that we should try to fix that in the future!

Reply:
So why previously this was one line and now two?

Reply:
Ah right. Sorry I responded to your question but not to the actual issue. It was because I removed `**kwargs` from `Raster.load()`. I've taken it back in a commit, so I will resolve this comment soon.

Reply:
Fixed in 5d7fcfc